### ISSUE TITLE
Create/Delegate a staking account

### ISSUE THREAD

**Issue by haianhltr** [haianhltr/stake_SOL#1]

**Comment by haianhltr** [comment]
Delegating stakes to validators means I trust this validator to accurately verify transactions on this blockchain. If a validator successfully writes accurate transactions without censorship, then the validator gets the reward, you will also get the reward because you stake your SOL with that validator. However, if the validator writes fraudulent transactions, you and the validator risk losing some SOL. 

In conclusion: Delegating your stake rewards people who protects the integrity of the Solana Blockchain.
